sphinx/environment/__init__.py

```diff
@@ -72,7 +72,7 @@
 # or changed to properly invalidate pickle files.
 #
 # NOTE: increase base version by 2 to have distinct numbers for Py2 and 3
-ENV_VERSION = 52 + (sys.version_info[0] - 2)
+ENV_VERSION = 53 + (sys.version_info[0] - 2)
```

Comment:
According to the comment above it should be increased by 2, so 54 instead of 53.

Otherwise Sphinx 1.7.9 with Python 2 will have the same version as Sphinx 1.7.8 with Python 3.

Reply:
oh, you're right. I'd forgotten it...